src/frontend/admin/src/hooks/useCertificateDefinitions/useCertificateDefinitions.ts

```diff
@@ -45,8 +45,9 @@ const messages = defineMessages({
   },
 });
 
```

Comment:
Global: I find the default dropdown misleading, it made me think that there were no options at all. What about "Please type ..."

<img width="564" alt="Capture d’écran 2024-02-14 à 17 58 45" src="https://github.com/openfun/joanie/assets/9628870/d38a1ffc-b5c0-4b05-895c-da17bf4e5bb4">